src/routes/__layout.svelte

```diff
@@ -1,33 +1,37 @@
 <script context="module" lang="ts">
-  import Nav from "$lib/components/Nav.svelte";
-  import Footer from "$lib/components/Footer.svelte";
+  import Nav from "$components/Nav.svelte";
+  import Footer from "$components/Footer.svelte";
+  import type { Load } from "@sveltejs/kit";
 
-  export async function load({ page, fetch }) {
+  export const load: Load = async ({ fetch }) => {
     const res = await fetch("/server/project-semesters.json");
 
     const semesters: string[] = await res.json();
 
-    const { path } = page;
-
-    return { props: { semesters, path } };
-  }
+    return { props: { semesters } };
+  };
 </script>
 
 <script lang="ts">
   import * as ackeeTracker from "ackee-tracker";
   import { beforeUpdate } from "svelte";
+  import "../app.css";
```

Comment:
If we do go ahead with not inlining global CSS in `app.html`, wouldn't it be more idomatic to add it to the style tag in this file?

Reply:
Addressed in the other comment, but the short answer is it's personal preference I guess? If we put it in the style tag we'd need to add `:global` prefixes to a bunch of the selectors. But, it would be easier to find in there.